simple_email_confirmation/admin.py

```diff
@@ -1,10 +1,11 @@
 from django.contrib import admin
 
-from .models import EmailAddress
+from .models import get_email_address_model
 
 
 class EmailAddressAdmin(admin.ModelAdmin):
     list_display = ('user', 'email', 'key', 'set_at', 'confirmed_at')
     search_fields = ('email', 'key')
 
-admin.site.register((EmailAddress,), EmailAddressAdmin)
+
+admin.site.register((get_email_address_model(),), EmailAddressAdmin)
```

Comment:
This does not need to be a tuple.

```suggestion
admin.site.register(get_email_address_model(), EmailAddressAdmin)
```

Reply:
Right, I never use a tuple in my admins, but it was a tuple before I made the change so I left it. I think it reads cleaner without the tuple so I will change it.